CalibMuon/DTCalibration/scripts/lcg-lsR.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
please give this a dt specific name (or perhaps its already provided by an wlcg environment and should not be in CMSSW?)